io.missioncenter.MissionCenter.json

```diff
@@ -0,0 +1,169 @@
+          "dest": "subprojects/nvtop-9a8458b541a195a0c5cadafb66e240962c852b39"
+        },
+        "cargo-sources.json"
+      ],
+      "modules": [
+        {
+          "name": "eudev",
+          "buildsystem": "autotools",
+          "config-opts": [],
+          "cleanup": [
+            "/bin",
+            "/sbin",
+            "/etc",
+            "/lib/udev"
+          ],
+          "sources": [
+            {
+              "type": "git",
+              "url": "https://github.com/eudev-project/eudev.git",
+              "commit": "39979ddf46e75d1b75bf381e1c73914c226c4302"
+            }
+          ]
+        },
+        {
+          "name": "musl-libc",
+          "buildsystem": "autotools",
+          "sources": [
+            {
+              "type": "archive",
+              "url": "https://musl.libc.org/releases/musl-1.2.4.tar.gz",
+              "sha256": "7a35eae33d5372a7c0da1188de798726f68825513b7ae3ebe97aaaa52114f039"
+            }
+          ]
+        },
+        {
+          "name": "dmidecode",
+          "buildsystem": "simple",
+          "build-commands": [
+            "make CC=musl-gcc CFLAGS=-static LDFLAGS=-static install-bin prefix=/app",
+            "install -v -p -m 755 /app/sbin/dmidecode /app/bin/dmidecode"
+          ],
+          "sources": [
+            {
+              "type": "archive",
+              "url": "https://git.savannah.nongnu.org/cgit/dmidecode.git/snapshot/dmidecode-3-5.tar.gz",
+              "sha256": "8b1465a1afa71f9ffb0206ba5ffc435c46c0281d089552578f600879728a1b23"
+            }
+          ]
+        },
+        {
+          "name": "proxy",
+          "buildsystem": "simple",
+          "build-options": {
+            "append-path": "/run/build/proxy/bin",
+            "env": {
+              "RUST_BACKTRACE": "full",
+              "RUSTUP_HOME": "/run/build/proxy",
+              "CARGO_HOME": "/run/build/proxy/cargo"
+            }
+          },
+          "build-commands": [
+            "cd src/proxy && env RUSTFLAGS='-C target-feature=+crt-static' cargo build --offline --release --target x86_64-unknown-linux-musl",
+            "install -v -p -m 755 src/proxy/target/x86_64-unknown-linux-musl/release/proxy /app/bin/missioncenter-proxy",
+            "rm -rf src/proxy/target"
+          ],
+          "sources": [
+            {
+              "type": "git",
+              "url": "https://gitlab.com/mission-center-devs/mission-center.git",
+              "tag": "v0.2.1",
+              "commit": "721d69a7a25f0903512d41e351d10db9db179928"
+            },
+            {
+              "type": "archive",
+              "url": "https://gitlab.com/mission-center-devs/build-tools/-/raw/main/rust-musl-1.70.0.tar.zst",
+              "sha256": "fcc101cd0af292d94e090b79517078c704201adff77817af0f654ca32f086f5b"
+            },
+            "cargo-sources-proxy.json"
+          ]
+        },
+        {
+          "name": "remove-leftover-headers-and-libraries",
```

Comment:
there are cleanup rule for this.

Reply:
Removed this module completely, added a post-install step for the `proxy` module.
The files installed by musl libc need to be cleaned up after `proxy` is built but before the main `missioncenter` module is built.
Using `cleanup` does not work here since the libraries were created by the `musl-libc` module. That being said, I moved everything that I could to the global cleanup.